### PR TITLE
update package '@azure/rest-api-specs-scripts' 0.4.0

### PR DESCRIPTION
**Pull request by ruowan**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by nschonni** [comment]
Might make sense to clean out the extra builds too https://github.com/Azure/azure-rest-api-specs/blob/20c172bfc51d6a38aca3cdf6b3faa7845e58cc11/preproduction-azure-pipelines.yml#L39-L112

File: preproduction-azure-pipelines.yml [hunk 1]
```diff
@@ -35,90 +35,3 @@ jobs:
...
-    displayName: 'npm install'
-    inputs:
-      verbose: false
-  - script: 'npm install -D @azure/rest-api-specs-scripts@0.4.0'
-  - script: 'scripts/install-dotnet.sh'
-    displayName: 'install .Net'
-  - script: 'npm run tsc && node scripts/momentOfTruth.js && node scripts/momentOfTruthPostProcessing.js'
-    displayName: 'LintDiff'
-
-- job: "Avocado"
```
Comment (nschonni) [inline preproduction-azure-pipelines.yml]: Did you mean to delete this one?

**Comment by ruowan** [comment]
@NullMDR @michaeljqzq Update `@azure/rest-api-specs-script` version and clean up preproduciton pipeline.

File: preproduction-azure-pipelines.yml (hunk 1 above)
Comment (ruowan) [reply preproduction-azure-pipelines.yml]: Sorry I didn't notice avocado PR has been merged into master. I will fix it.

**Review by michaeljqzq — APPROVED** [review]

**Pull request merged**